contracts/erc1155/MintableERC1155.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC1155/presets/ERC1155PresetMinterPauser.sol";
+
+contract MintableERC1155 is ERC1155PresetMinterPauser {
+    mapping(uint256 => string) public _tokenURIs;
+
+    event Airdrop(address caller, uint256 tokenId, address[] recipients, uint256[] amounts);
+
+    constructor() ERC1155PresetMinterPauser("") {}
```

Comment:
The `Solhint` fails here - should we add a comment override or add a no-op to remove this issue?

Reply:
This code might change but FYI you can silence warnings if they're not useful via something like 
`// solhint-disable-next-line avoid-low-level-calls`

Reply:
Added the lint disable here https://github.com/across-protocol/contracts-v2/pull/262/commits/b5831e7c0f750d3abcb0e83edb6177c48e105118